### ISSUE TITLE
Stats::View::Google::Graph lose first statistics

### ISSUE THREAD

**Issue by eniocarboni** [eprints/irstats2#69]
I seem that Stats::View::Google::Graph lose a few pieces of statistics.
I'll explain, in fact in some cases the first valid month (or year or day based on date resolution) is lost
You can see an example on the same graph with a different range:
http://wrap.warwick.ac.uk/cgi/stats/report/eprint/74377 (all seem ok)
http://wrap.warwick.ac.uk/cgi/stats/report/eprint/74377?from=2015%2F10%2F01&to=2016%2F02%2F29 (we lose Nov 2015)

Activity Overview (Stats::View::KeyFigures) is always ok

I suppose and tested that in Stats::View::Google::Graph by deleting line 120

`$i++ if( $i == 0 );`

we solve the problem.

Summing up the problem is that you lose the statistics for the first month if the graph does not start with that month.

**Comment by jiadiyao** [comment]
Same issue were experienced by OCADU, and the above line were commented out since March 2016 to fix the discrepancy. It has been running normally since.
